Entities/AEmitter.cpp

```diff
@@ -38,9 +38,8 @@ void AEmitter::Clear()
     m_WasEmitting = false;
     m_EmitCount = 0;
     m_EmitCountLimit = 0;
-	m_MinThrottleRange = 1.0F;
-	m_MaxThrottleRange = 1.0F;
-    m_ThrottleRangeRedefined = false;
```

Comment:
I think you've forgotten the point of this branch. Your changes before to humans and crabs made it so jetpacks would lose strength if they were running out of fuel, which we didn't want happening to mods by default, unless modders defined throttle multipliers.
The "useless" flag you deleted was what kept this from happening, cause there was a check in AHuman and ACrab create that says "if jetpack throttle ranges haven't been defined, default them to 0 to avoid compatibility problems". 
Also, not to be a dick but you 100% didn't run this code to see if it works :ju:. I know cause you forgot to remove this aforementioned check (or update comments that say `if throttle range values have been defined`) or fix stuff using old method names that you changed.

Reply:
The default value for both of the new throttle multipliers is 1, which means that throttle will have no effect on emission rate. This means that jetpacks will function as before, unless the property is changed.

Reply:
Okay, I'll wait til you clean up then. I know the branch shuffling stuff was a pain, so thanks for dealing with it.

Reply:
The math of throttleFactor stuff deals with this in a slightly roundabout way, so it's fine as-is.